crates/ibc/src/core/ics03_connection/msgs/conn_open_try.rs

```diff
@@ -27,22 +25,20 @@ pub const TYPE_URL: &str = "/ibc.core.connection.v1.MsgConnectionOpenTry";
 ///
 #[derive(Clone, Debug, PartialEq, Eq)]
 pub struct MsgConnectionOpenTry {
-    pub previous_connection_id: Option<ConnectionId>,
     pub client_id: ClientId,
     pub client_state: Any,
     pub counterparty: Counterparty,
     pub counterparty_versions: Vec<Version>,
-    pub proofs: Proofs,
+    pub proof_connection_end: CommitmentProofBytes,
+    pub proof_client_state: CommitmentProofBytes,
+    pub proof_consensus_state: CommitmentProofBytes,
+    pub proofs_height: Height,
+    pub consensus_height: Height,
     pub delay_period: Duration,
     pub signer: Signer,
-}
 
-impl MsgConnectionOpenTry {
-    /// Getter for accessing the `consensus_height` field from this message.
-    /// Returns `None` if this field is not set.
-    pub fn consensus_height(&self) -> Option<Height> {
-        self.proofs.consensus_proof().map(|proof| proof.height())
-    }
+    /// deprecrated. Only kept here for proper conversion to/from the raw type
+    previous_connection_id: String,
```

Comment:
Should we add the `#[deprecated(since = "0.20.x")]` attribute here?

Reply:
Note: Marked deprecated and made the field public again
[192f903](https://github.com/cosmos/ibc-rs/pull/170/commits/192f9039589ae5664f4915f0c3f4fc745f47a691)